wa/starter/src/main/java/org/apache/syncope/wa/SyncopeCoreNotReadyException.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Is this exception really required? How are existing components in WA managing the fact that WARestClient is  not ready yet?

Reply:
I think @mmoayyed will be able to help us about this question!

Reply:
ATM in the other places where we check whether the client is ready or not we just have logging statements, no exceptions!

Reply:
Strictly speaking, this is not required here because we'd be making an assumption that core is in fact ready by the time this method is called (and is only called once the app is ready). If by then core is not ready, we should likely consider that a catastrophic event and end the app, specially because this (metadata, etc) is not the sort of thing one can ignore unlike other places. So perhaps we can just return `false` here, but...

..more importantly, I see the catch clause that traps `Exception`, and by extension this one, just logs a warning. I don't think that's right; at the point in the lifetime of the app when this method is called, core must be ready. Its unavailability should halt the app and present an error (more than a warning).

Reply:
@mmoayyed What do you mean with "So perhaps we can just return `false` here"?
According to your clarification I guess that one option could be to pull out the `getSyncopeClient()` from the `try/catch` and make it throw the blocking exception in order to block the execution.
But about the `SyncopeCoreNotReadyException`, do you guys want to keep it or just throw a `RuntimeException` with `Syncope core is not yet ready to access requests` as message?